src/main/resources/config/config.properties

```diff
@@ -0,0 +1,4 @@
+url= http://automationpractice.com/index.php
```

Comment:
rename it to a more suggestive variable e.g. **homeUrl**

Reply:
Done